src/core/lombok/Builder.java

```diff
@@ -120,6 +120,13 @@
 	@Retention(SOURCE)
 	public @interface Default {}
 
+	/**
+	 * The field annotated with {@code @Exclude} will be excluded from the builder.
```

Comment:
Maybe we should consider adding a legend to indicate that Objects will be set to null and primitives to its defaul value

Reply:
yes, i can do that. on it

Reply:
done